### PR TITLE
#14: Update regression test result upload

### LINKED ISSUE #14: Adapt to new regression test setup
Uploading of the regression test result has changed. We need to adapt to the new process.

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #14

### CODE REVIEW

File: .ci/retry.sh [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
+set -euo pipefail
 
-numRetries=3
-retryDelay=1
+readonly numRetries=3
```
Comment (redcatbear) [inline .ci/retry.sh:4]: Bash coding conventions usually use lower snake case. But that is not so important.
https://github.com/easybash/bash-coding-style-guide#variable

File: .ci/upload_regression_test_result.sh [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+readonly jdbc_client_dir="$target_dir/EXASOL_JDBC-$jdbc_client_version"
+
+mkdir -p "$target_dir"
+
+cd "$target_dir"
+
+if [[ ! -d "$jdbc_client_dir" ]]; then
+    echo "Downloading JDBC client from $jdbc_client_url"
+    curl $jdbc_client_url -o "$jdbc_client_file"
+    echo "$jdbc_client_sha512_sum  $jdbc_client_file" | sha512sum -c
```
Comment (redcatbear) [inline .ci/upload_regression_test_result.sh:21]: :+1:

File: doc/developers_guide/developers_guide.md [hunk 3]
```diff
@@ -2,19 +2,23 @@
...
+
+Integration tests need access to Google Cloud. To configure this, follow [these instructions](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating) to create a service account key and save it to file `google-key.json`.
+
 ## Running Regression Test
 
 This project contains some regression tests to monitor the performance cross releases. To run them locally use:
 
 ```shell
-mvn verify -P regressionTests
+mvn verify -PregressionTests
```
Comment (redcatbear) [inline doc/developers_guide/developers_guide.md:14]: Shouldn't this do the exact same thing?

**Review by redcatbear — APPROVED** [review]

File: doc/developers_guide/developers_guide.md (hunk 3 above)
Comment (kaklakariada) [reply doc/developers_guide/developers_guide.md:14]: Good question. The workflow file uses `-PregressionTests`.

File: .ci/retry.sh (hunk 1 above)
Comment (kaklakariada) [reply .ci/retry.sh:4]: I will keep this in mind for the future.

**Review by redcatbear — APPROVED** [review]

**Pull request merged**